core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathUtil.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.expression.operator.arthmetic;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class MathUtil {
+
+  /**
+   * Truncates a number to required decimal places.
+   *
+   * @param numberToTruncate number to be truncated
+   * @param numberOfDecimals required decimal places
+   * @return truncated number as {@link BigDecimal}
+   */
+  public static BigDecimal truncateNumber(double numberToTruncate, int numberOfDecimals) {
```

Comment:
Please, add unit tests with different `numberOfDecimals` values. The test should use another function to validate the result, for example, `substring`:
```java
assertEquals(value.toString().subString(...), truncateNumber(value, scale).toString())
```